3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentRegistry.scala

```diff
@@ -184,6 +187,15 @@ case class EnrichmentRegistry(private val configs: EnrichmentMap) {
    */
   def getCampaignAttributionEnrichment: Option[CampaignAttributionEnrichment] = 
     getEnrichment[CampaignAttributionEnrichment]("campaign_attribution")
+  
+  /**
+   * Returns an Option boxing the UserAgentUtilsEnrichment
+   * config value if present, or None if not
+   *
+   * @return Option boxing the UserAgentUtilsEnrichment instance
+   */
+  def getUserAgentUtilsEnrichment: Option[UserAgentUtilsEnrichment] = 
+    getEnrichment[UserAgentUtilsEnrichment]("user_agent_utils_config") 
```

Comment:
"user_agent_utils" here, not _config, as this is the name of the overall enrichment...